File: Administrativo/spring-erp-servicio/net/royal/spring/rrhh/dominio/filtros/FiltroPuestoempresa.cs
using net.royal.spring.framework.core.dominio;
using System;
using System.Collections.Generic;
using System.Text;

namespace net.royal.spring.rrhh.dominio.dto
{
    public class FiltroPuestoempresa
    {
       public Int32? codigo { get; set; }
       public String nombre{ get; set; }

        public ParametroPaginacionGenerico paginacion { get; set; }

        public FiltroPuestoempresa()
        {
            paginacion = new ParametroPaginacionGenerico();
        }

    }
}

File: Administrativo/spring-erp-web/SpringNg/src/app/erp_module/shared/selectores/puestoempresa/dominio/FiltroPuestoempresa.ts
import { ParametroPaginacionGenerico } from "../../../dominio/ParametroPaginacionGenerico";


export class FiltroPuestoempresa {

    constructor() {
        this.paginacion = new ParametroPaginacionGenerico();
    }
    paginacion: ParametroPaginacionGenerico;

    codigo: number;
    nombre: string;

    
}
